test/pkey_name_dup.c

```diff
@@ -0,0 +1,63 @@
+#include <stdlib.h>
```

Comment:
It would make sense to add this to `evp_test.c` so it covers all key types and avoids adding yet more tiny tests.

Reply:
On the other hand `evp_test.c` is rather a monstrous kitchen sink, and as mentioned above, it is not clear we need to test this across all algorithms. But if consensus it to take that route, I am not opposed.